environments/india/inventory.ini

```diff
@@ -17,11 +17,11 @@
 [celery1]
 10.203.10.13 hostname=celery1-india ufw_private_interface=ens5 ansible_python_interpreter=/usr/bin/python3 swap_size=4G
 [couch5]
-10.203.40.178 hostname=couch5-india ufw_private_interface=ens5 ansible_python_interpreter=/usr/bin/python3 datavol_device=/dev/nvme1n
+10.203.40.178 hostname=couch5-india ufw_private_interface=ens5 ansible_python_interpreter=/usr/bin/python3 datavol_device=/dev/sdf datavol_device1=/dev/sdf is_datavol_ebsnvme=yes
```

Comment:
Why is `datavol_device1` defined here? It also appears to be configured automatically in [src/commcare_cloud/ansible/roles/ecryptfs/defaults/main.yml](https://github.com/dimagi/commcare-cloud/pull/2991/files#diff-8a1586d90c304abac8ef7fc5ea06c793)

Reply:
I'm defining it here to override the automatically configured default. The default would have it be `/dev/sdf1`, but I want it to be `/dev/sdf`. I'm pretty sure `datavol_device1` is a bad name but I'm not sure what to call it—I thought of `datavol_partition`, but it doesn't actually have to be a disk partition, it can be any device.